src/material-experimental/mdc-form-field/form-field.ts

```diff
@@ -584,7 +584,9 @@ export class MatFormField implements AfterViewInit, OnDestroy, AfterContentCheck
     if (this._control) {
       let ids: string[] = [];
 
-      if (this._control.userAriaDescribedBy) {
+      if (this._control.userAriaDescribedBy &&
+        typeof this._control.userAriaDescribedBy === 'string') {
```

Comment:
I don't quite follow the change here and in the other `form-field.ts`. I don't think this code in particular has been touched in a while.

Reply:
This was added in this PR https://github.com/angular/components/pull/19587

Reply:
Hmm, I see. Maybe we should change it to `(this._control.userAriaDescribedBy + "").split(' ')` so it's guaranteed to always be a string?

Reply:
I think it's ok to leave it as is (unless you feel strongly about this cause then I have no problem making this change) because I'm going to track down where the gap in our type checking is for this and close it. That way we can safely assume `userAriaDescribedBy` is a string and we can go back to the original way this conditional looked

Reply:
It's fine, I don't feel strongly against it.